action_server/src/sema4ai/action_server/_cli_impl.py

```diff
@@ -553,54 +553,63 @@ def _main_retcode(
         download_rcc(target=download_args.file, force=True)
         return 0
 
-    if command == "package":
-        from sema4ai.action_server.package._package_build_cli import (
-            handle_package_command,
-        )
+    from sema4ai.action_server._download_rcc import download_rcc
+    from sema4ai.action_server._rcc import initialize_rcc
+    from sema4ai.action_server._session import initialize_session
 
-        return handle_package_command(base_args)
+    with initialize_rcc(download_rcc(), None) as rcc:
+        initialize_session(rcc)
 
-    if command == "env":
-        from sema4ai.action_server.env import handle_env_command
+        if command == "package":
+            from sema4ai.action_server.package._package_build_cli import (
+                handle_package_command,
+            )
 
-        return handle_env_command(base_args)
+            return handle_package_command(base_args)
 
-    if command == "cloud":
-        from sema4ai.action_server._actions_cloud import handle_cloud_command
+        if command == "env":
+            from sema4ai.action_server.env import handle_env_command
 
-        return handle_cloud_command(base_args)
+            return handle_env_command(base_args)
```

Comment:
Since `rcc` is already initialized here, I guess it should be passed to the `handle_env_command` now (maybe `initialize_rcc` should be changed to check if `_rcc` is not `None` and if it is raise an error to avoid initializing multiple times).

Reply:
Very good point 👍 did the changes.